API.md

```diff
@@ -18,7 +18,58 @@ _Returns:_ A string representing a bearer token that can be used to carry out op
 
 ### Scheme: OpenID Connect (OIDC)
 
-Coming soon ...
+For authenticating with OpenID Connect, there are two methods:
+
+- Resource Owner Password Credentials Grant
+- Authorization Code Grant
+
+#### Resource Owner Password Credentials Grant
+
+The core advantage of this grant is the flexibility in how it may be applied.
+
+Following is a brief example of authenticating from the command line.
+
+##### Request
+```bash
+username_and_password=$(echo -n "<username>:<password>" | base64 -)
+curl -X POST -H "Authorization: iRODS $username_and_password" \
+    http://localhost:<port>/irods-http-api/<version>/authenticate
+```
+
+##### Response
+A string representing a bearer token that can be used to carry out operations as the authenticated user.
+
+#### Authorization Code Grant
+
+Using this grant requires a bit more work to extract the token.
+Authentication is done in the browser.
+
+##### Request
```

Comment:
We don't have a `##### Response` section for this and it's not clear where it should be added.

Maybe this header should become `##### Request / Response`. What do you think?

Reply:
Sure. Let's go with your suggestion.

Reply:
Done.